recipes/libtensorlight/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "3.0.1" %}
+
+package:
+  name: libtensorlight
+  version: {{ version }}
+
+source:
+  url: https://github.com/adc-connect/libtensor/archive/v{{ version }}.tar.gz
+  sha256: 1ec596bbd160b4df413bccb42e88d876e098aa62899f6b1bb50003d5552bf9cb
+
+build:
```

Comment:
This also needs a fitting run_exports.

Reply:
I'm always uncertain how to specify `run_exports`, also after re-reading [docs](https://conda-forge.org/docs/maintainer/pinning_deps.html#specifying-run-exports) again...

Would
```yaml
run_exports:
    - {{ pin_subpackage('libtensorlight', max_pin='x.x') }}
```
be fitting?